CustomApps/lyrics-plus/Translator.js

```diff
@@ -98,10 +98,20 @@ class Translator {
 		}
 	}
 
+	/**
+	 * Async wrapper of `setTimeout`.
+	 *
+	 * @param {number} ms
+	 * @returns {Promise<void>}
+	 */
+	static async #wait(ms) {
```

Comment:
```suggestion
	static async #sleep(ms) {
```
I would also move this function to the bottom of the class